lib/vagrant-vbguest/installers/windows.rb

```diff
@@ -50,40 +54,50 @@ def install(opts = nil, &block)
       end
 
       def running?(opts = nil, &block)
-        communicate.test("get-service VBoxService", opts, &block)
+        # Windows always needs to reboot after installation
+        false
```

Comment:
But this will also trigger a false negative result when running the first checks. Like when the correct version is installed and running.

We have a similar problem on linux, where in some cases we need a reboot (or at least virtual box recommends it) to properly activate a never version of kernel modules.
As a starter I'd rather keep the `running?` checks as they are (doing what they are supposed to do) and add another method and probably vm-config (`reboot_after_install?` or something like that) to force a reboot after installation.

Reply:
I've reinstated `running?` with a better check, and just directly rebooted from inside `install` as this makes the most sense (I could have split this to a separate function but it's a single method call).